app/bundles/CoreBundle/Entity/Notification.php

```diff
@@ -31,7 +31,7 @@ class Notification
     protected $header;
 
     /**
-     * @var string|null
+     * @var string
```

Comment:
Defined in line 85 as not nullable

Reply:
Same as https://github.com/mautic/mautic/pull/12262#discussion_r1185921179

Reply:
solved in commit ca37c0d4dbd77b237bbf1faadddc0dfdbfc0efac

Reply:
One thing that we could do instead of skipping the required check is to require the required properties in the entity constructors and that would ensure they are really required. But we should do that in another PR as it will require more refactoring.